<launch>

    <node pkg    = "robot_pose_ekf"
          type   = "robot_pose_ekf"
          name   = "robot_pose_ekf"
          output = "screen">

        <param name = "output_frame"          value = "wheel_ekf_init"/>
        <param name = "base_footprint_frame"  value = "wheel_ekf_pose"/>
        <param name = "odom_used"             value = "true"/>
        <param name = "imu_used"              value = "true"/>
        <param name = "vo_used"               value = "false"/>

        <remap from = "/odom"     to = "/robot_odom"/>
        <remap from = "/imu_data" to = "/robot_imu"/>
        <remap from = "/robot_pose_ekf/odom_combined"
                 to = "/robot_pose_ekf/pose_combined"/>

    </node>

    <node pkg    = "robot_pose_ekf"
          type   = "pose_to_odom.py"
          name   = "pose_to_odom_node"
          output = "screen"/>

    <node pkg    = "robot_pose_ekf"
          type   = "pitch_to_z_axis.py"
          name   = "pitch_to_z_axis_node"
          output = "screen"/>

</launch>

